Sources/StreamChatSwiftUI/DefaultViewFactory.swift

```diff
@@ -95,6 +96,18 @@ extension ViewFactory {
         )
     }
     
+    public func makeChannelAvatarView(
+        for channel: ChatChannel,
+        showOnlineIndicator: Bool,
+        size: CGSize
```

Comment:
It will be easy to break this in the future. There's is a chance we might introduce other stuff. Better to create a `ChannelAvatarViewOptions` struct, something like:

```swift
struct ChannelAvatarViewOptions {
    var showOnlineIndicator: Bool
    var size: CGSize
}
```

Then:
```swift
public func makeChannelAvatarView(
    for channel: ChatChannel,
    with options: ChannelAvatarViewOptions
)
```

We have something similar on UIKit

Reply:
good point, let me do that